src/lesson7/javaBeginnerChapter2/task24/Time.java

```diff
@@ -0,0 +1,61 @@
+package lesson7.javaBeginnerChapter2.task24;
+
+public class Time {
+    private int seconds;
+    private int minutes;
+    private int hours;
+
+    public Time(int seconds) {
+        this.seconds = seconds;
+    }
+
+    public Time(int seconds, int minutes, int hours) {
+        this.seconds = seconds;
```

Comment:
Implement logic for conversion one type to another. 
Now, for example, I can create Time(32425, 3254254, 1321314) and it'll be stored exactly this way. But you should to store seconds with value less than 60, and minutes too

Reply:
fixed